Telegram-bot-example/Sources/Telegram-bot-example/main.swift

```diff
@@ -4,7 +4,7 @@ import TelegramVaporBot
 var env = try Environment.detect()
 try LoggingSystem.bootstrap(from: &env)
 let eventLoop: EventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: System.coreCount * 4)
-let app: Application = .init(env, Application.EventLoopGroupProvider.shared(eventLoop))
+let app: Application = try! await Application.make(env, Application.EventLoopGroupProvider.shared(eventLoop))
```

Comment:
Hi, why did you make a forceunwrap try! ?

Reply:
Removed force from force try.

Reply:
Thanks